doc/rados/operations/crush-map.rst

```diff
@@ -1198,8 +1201,10 @@ profile.  Those are:
  * ``argonaut``: the legacy values supported by the original argonaut release
  * ``bobtail``: the values supported by the bobtail release
  * ``firefly``: the values supported by the firefly release
- * ``optimal``: the current best values
- * ``default``: the current default values for a new cluster
+ * ``optimal``: the best (ie optimal) values of the current version of Ceph
+ * ``default``: the default values of a new cluster installed from
+   scratch (these values, which depend on the current version of Ceph,
+   are hard coded and are generally a mix of optimal and legacy values)
```

Comment:
i would say "these values, which depend on the current version of Ceph, generally match the previous LTS release, or the most recent release for which we generally except more users to have up to date clients for"